Add headers to KNOWN_ISSUES (#32)
IBM/cloudant-python-sdk: KNOWN_ISSUES.md

<!-- Template substitution for language specific content -->
<!-- ## SPLIT MARKER ## -->
## Cloudant SDK for Python

> ### Bodies containing the `headers` parameter

Reviewer: Nit

```suggestion
### Request bodies containing the `headers` parameter
```

Author: Fixed in d1ac0de.

The `headers` dict is reserved for the API request headers.
In order to avoid collision between the request headers and the `headers` parameter
that is contained by the request body, use the `headers_` as
the request body `headers` parameter.

> E.g. for applying the `headers` parameter of your replicate

Reviewer: Nit
```suggestion
E.g. to set the `headers` parameter of your replication
```

Author: Fixed in d1ac0de.
